src/backend/src/utils/projects.utils.ts

```diff
@@ -117,6 +117,10 @@ export const projectTransformer = (
     bomLink: project.bomLink ?? undefined,
     rules: project.rules,
     duration: calculateDuration(project.workPackages),
+    endDate: calculateEndDate(
```

Comment:
we might need a new function for project end date calculation because this doens't seem right